src/core_plugins/kibana/public/management/sections/indices/add_data_steps/parse_csv_step/lib/validate_headers.js

```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash';
+
+export default function validateHeaders(fields) {
+  if (!_.isArray(fields)) {
+    throw new Error('first argument must be an array');
+  }
```

Comment:
Yeah. This is a minor one. And I think it is worth considering in the future if the validateHeaders module grows more than a couple validation checks.